vllm/config.py

```diff
@@ -186,6 +187,7 @@ def __init__(
                                     code_revision, rope_scaling, rope_theta,
                                     config_format)
         self.hf_text_config = get_hf_text_config(self.hf_config)
+        self.bert_config = self._get_bert_config()
```

Comment:
Could we call this something more general than "bert_config"? Maybe "encoder_config"?